lib/screens/send_transaction/send_vtt_screen.dart

```diff
@@ -76,10 +80,17 @@ class CreateVttScreenState extends State<CreateVttScreen>
       setState(() {
         currentStepIndex += 1;
         stepSelectedItem = stepListItems[currentStepIndex];
+        vttStepsFormKey.currentState!.setState(() {
+          vttStepsFormKey.currentState!.selectedItem =
+              _localizedVttSteps[stepSelectedItem]!;
+        });
       });
     }
   }
 
+  Map<Enum, String> get _localizedVttSteps =>
+      localizeEnum(context, VTTsteps.values, _localization.vttSendSteps);
```

Comment:
I think I will be easier to read if we directly create the map in this file, so you can see the key/value.

Reply:
i do not think this is not possible, since the values are dynamic and retrieved from a localization file, we cannot have a static map.